tests/shared/src/test/scala/org/threeten/bp/TestClock.scala

```diff
@@ -45,7 +45,8 @@ object TestClock {
 
     def getZone: ZoneId = zone
 
-    def withZone(timeZone: ZoneId): Clock = new TestClock.MockInstantClock(millis, timeZone)
+    override def withZone(timeZone: ZoneId): Clock =
```

Comment:
Is this required for java 17?

Reply:
I don't know how the build passed without this. Not related to Java 17 at all. I had to do this also for Scala Native PR.